### PR TITLE
clean-up of styling and code organization TM-3184

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
I don't think i broke anything... checked:

AIH-card
AIH-row
Emp Search-card
Emp Search-row

### CODE REVIEW

**Comment by SidelineCory24** [comment]
All are still working as expected:
- [ ] AIH-card
- [ ] AIH-row
- [ ] Emp Search-card
- [ ] Emp Search-row

File: src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx [hunk 1]
```diff
@@ -254,7 +254,7 @@ const AgendaItemLegs = props => {
   const tableData$ = isCard ? filter(tableData, 'cardView') : tableData;
 
   return (
-    <div className="ai-history-card-legs">
+    <div className={`${isAIHView ? 'ai-history-card-legs' : 'aim-legs'}`}>
```
Comment (SidelineCory24) [inline src/Components/Agenda/AgendaItemLegs/AgendaItemLegs.jsx:257]: 😎

File: src/sass/_agendaItemMaintenance.scss [hunk 2]
```diff
@@ -32,13 +32,14 @@
 
   .expand-arrow {
     align-self: center;
+    color: $primary-blue;
```
Comment (SidelineCory24) [inline src/sass/_agendaItemMaintenance.scss:35]: 👀

File: src/sass/_employeeSearch.scss [hunk 3]
```diff
@@ -414,435 +414,3 @@
   }
 }
 
-.agenda-item-history-container {
```
Comment (SidelineCory24) [inline src/sass/_employeeSearch.scss:417]: 😱 all gone

File: src/sass/_agendaItemHistory.scss [hunk 4]
```diff
@@ -0,0 +1,432 @@
+.agenda-item-history-container {
```
Comment (SidelineCory24) [inline src/sass/_agendaItemHistory.scss:1]: 🥹 all back

File: src/sass/_navTabs.scss [hunk 5]
```diff
@@ -10,11 +10,18 @@
...
       font-size: 13px;
       padding: 5px 10px;
       text-align: center;
     }
+    // special styling for first tab
+    #asgh-tab-container {
+      .tab{
+        border-left: 0;
+      }
+    }
```
Comment (SidelineCory24) [inline src/sass/_navTabs.scss:24]: nice catch!

**Comment by SidelineCory24** [comment]
image

**Comment by SidelineCory24** [comment]
With the background color being the same now did we want this much space in-between the legs and research pane?
image

**Comment by elizabeth-jimenez** [comment]
> With the background color being the same now did we want this much space in-between the legs and research pane? Gives room for anything in the future if needed. image

The possibilities are endless 🌌

Nice catch! That's def not intentional lol - i often have devtools open and that made the gap much smaller 😭  will update

**Comment by SidelineCory24** [comment]
> > With the background color being the same now did we want this much space in-between the legs and research pane? Gives room for anything in the future if needed. image
> 
> The possibilities are endless 🌌
> 
> Nice catch! That's def not intentional lol - i often have devtools open and that made the gap much smaller 😭 will update

same, I didn't notice it until I closed devtools either 😂

**Comment by elizabeth-jimenez** [comment]
> > > With the background color being the same now did we want this much space in-between the legs and research pane? Gives room for anything in the future if needed. image
> > 
> > 
> > The possibilities are endless 🌌
> > Nice catch! That's def not intentional lol - i often have devtools open and that made the gap much smaller 😭 will update
> 
> same, I didn't notice it until I closed devtools either 😂

What are the specs for this?

**Comment by elizabeth-jimenez** [comment]
> > > > > With the background color being the same now did we want this much space in-between the legs and research pane? Gives room for anything in the future if needed. image
> > > > 
> > > > 
> > > > The possibilities are endless 🌌
> > > > Nice catch! That's def not intentional lol - i often have devtools open and that made the gap much smaller 😭 will update
> > > 
> > > 
> > > same, I didn't notice it until I closed devtools either 😂
> > 
> > 
> > What are the specs for this?
> 
> Extending the legs some to close the spacing.

I meant screen size and how much it's zoomed in, if any

**Comment by SidelineCory24** [comment]
> > > > > > With the background color being the same now did we want this much space in-between the legs and research pane? Gives room for anything in the future if needed. image
> > > > > 
> > > > > 
> > > > > The possibilities are endless 🌌
> > > > > Nice catch! That's def not intentional lol - i often have devtools open and that made the gap much smaller 😭 will update
> > > > 
> > > > 
> > > > same, I didn't notice it until I closed devtools either 😂
> > > 
> > > 
> > > What are the specs for this?
> > 
> > 
> > Extending the legs some to close the spacing.
> 
> I meant screen size and how much it's zoomed in, if any

Discussed offline, zooming issue. No change needed

**Comment by SidelineCory24** [comment]
🤩
image

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

**Pull request merged**